x/bank/msgs.go

```diff
@@ -27,6 +28,49 @@ func (msg MsgSend) Type() string  { return "send" }
 
 // Implements Msg.
 func (msg MsgSend) ValidateBasic() sdk.Error {
+	if msg.FromAddress.Empty() {
+		return sdk.ErrInvalidAddress("No FromAddress included")
+	}
+	if msg.ToAddress.Empty() {
+		return sdk.ErrInvalidAddress("No ToAddress included")
```

Comment:
```suggestion
		return sdk.ErrInvalidAddress("missing to address")
```

Reply:
`missing recipient address` or `missing "to" address`?